delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ProjectsRoutes.scala

```diff
@@ -55,7 +54,7 @@ final class ProjectsRoutes(
     cr: RemoteContextResolution,
     ordering: JsonKeyOrdering,
     fusionConfig: FusionConfig,
-    contextShift: ContextShift[IO]
+    runtime: IORuntime
```

Comment:
You only need to import:
```
import cats.effect.unsafe.implicits.global
```
Where we have a call to an unsafe method

Reply:
The text you quoted the other day:

> Instead of calling unsafe methods directly, consider using cats.effect.IOApp, which runs your IO. If integrating with non-functional code or experimenting in a REPL / Worksheet, add the following import:
> `import cats.effect.unsafe.implicits.global`

Doesn't that imply that we should NOT do that in production?

Reply:
The implicit:
```
object implicits {
  implicit def global: IORuntime = IORuntime.global
}
```
Which is the same value as the one you pass down

I think what they meant is that you should not use it unless you get away from the holy path of functional code.
And akka falls into that category because you need to call `unsafeFuture`